### PR TITLE
Add aborghi to secret things

### PR DESCRIPTION
**Pull request by jwkaltz**

### CODE REVIEW

File: CONST_Makefile [hunk 1]
```diff
@@ -303,6 +303,7 @@ GPG_KEYS += 1DAFEE6A # Oliver Christen
 GPG_KEYS += C8A0FF72 # Wolfgang Kaltz
 GPG_KEYS += A510717D # Michael Künzli
 GPG_KEYS += A68C2F5F # Arnaud Morvan
+GPG_KEYS += 0545262E5AE04436C83B654D4CEC60DAC4F68132 # Andrea Borghi
```
Comment (sbrunner) [inline CONST_Makefile:305]: Please use short key as for other (8 last as I remember...)

**Review by sbrunner — APPROVED** [review]

File: CONST_Makefile (hunk 1 above)
Comment (jwkaltz) [reply CONST_Makefile:305]: Using long keys for everybody, as discussed in https://github.com/camptocamp/c2cgeoportal/pull/5540

**Comment by jwkaltz** [comment]
@sbrunner can I merge (despite CircleCI error)?

**Comment by sbrunner** [comment]
+1

**Pull request merged**